app/tests/spec/views/mixins/settings-panel-mixin.js

```diff
@@ -76,6 +76,30 @@ define(function (require, exports, module) {
         assert.isFalse(view.isPanelOpen());
       });
 
+      it('openPanel focuses the first input field or button if present', function () {
+        view.openPanel();
+        var input = $('.open input:first-of-type');
+        if (input.length > 0) {
+          // if input field is present, then that should have been focused
+          assert.equal(input[0], document.activeElement, "Input doesn't have focus");
+        } else {
+          // if button is present, then that should have been focused
+          var button = $('.open button.primary');
+          if (button.length > 0) {
```

Comment:
When you split these tests apart, make the DOM so that you *know* a button.primary is going to be there, and test accordingly. Get rid of the element check.